pom.xml

```diff
@@ -52,17 +52,20 @@
     <gitHubRepo>jenkinsci/${project.artifactId}-plugin</gitHubRepo>
     <jira-rest-client.version>5.2.7</jira-rest-client.version>
     <fugue.version>4.7.2</fugue.version>
-    <!-- jenkins -->
-    <jenkins.version>2.440.3</jenkins.version>
+    <!-- TODO: Remove when 2.472 is available in LTS -->
+    <jenkins.version>2.472</jenkins.version>
+    <!-- TODO JENKINS-73339 until in parent POM -->
+    <jenkins-test-harness.version>2254.vcff7a_d4969e5</jenkins-test-harness.version>
     <spotless.check.skip>false</spotless.check.skip>
+    <maven.compiler.release>17</maven.compiler.release>
```

Comment:
This is/should be coming from parent-pom, we don't want an override here

Reply:
@rantoniuk There is not yet a release of plugin parent POM that requires Java 17, but there will be one around the October timeframe, at which point this temporary code can be deleted. We aren't releasing a plugin parent POM that requires Java 17 right now because most of the ecosystem is still targeting Jenkins core releases that support Java 11.